Sources/Apollo/ApolloClient.swift

```diff
@@ -206,7 +215,7 @@ private func wrapResultHandler<Data>(_ resultHandler: GraphQLResultHandler<Data>
 }
 
 private final class FetchQueryOperation<Query: GraphQLQuery>: AsynchronousOperation, Cancellable {
-  let client: ApolloClient
+  weak var client: ApolloClient?
```

Comment:
Why does this need to be `weak` here?

Reply:
I think you're adding the operation to the client's queue
https://github.com/apollographql/apollo-ios/blob/509b2220c7a2e5dc5d0940b11361df0068ed4a0c/Sources/Apollo/ApolloClient.swift#L137-L138
so both the client and the operation will be retained here

Reply:
oh woof, good point.